pytrials/client.py

```diff
@@ -112,5 +114,18 @@ def get_study_fields(self, search_expr, fields, max_studies=50, fmt="csv"):
             else:
                 raise ValueError("Format argument has to be either 'csv' or 'json'")
 
+    def get_study_count(self, search_expr, fields=["NCTId"]):
```

Comment:
Another suggestion, write a docstring similar to the other ones you see above. It should have a title of what the function does,  a brief description (optional), arguments the function takes, and what it returns, as well as the errors it can raise.

Reply:
Hi there.  I did actually add a functional test for the new feature in test_client.py starting on line 66.  I didn't add any negative tests though.  I could think of one and add that as well as the docstring.  When I do that, should I cancel this pull request and create a new one?  I'm not sure of best practices, so appreciate any guidance.  Thanks!

Reply:
Oh, sorry, you're absolutely right about the test, I just overlooked it, thanks. No need to add any further tests, I'll probably do an overhaul on the testing suite before the next update, so don't worry about it. 

For adding the docstring, you can just edit the file, add the docstring as you see fit and push as a new commit. No need to create another pull request, since it's dealing with the same feature/update in general.